### DISCUSSION TITLE
Bug Fixes

### DISCUSSION THREAD

**Discussion by torchsmith** [elementor/elementor#23593] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Avoid so many bugs in Elementor.

### Describe the solution you'd like

Instead of releasing a new version of Elementor all the time with new features (and new bugs), why don't you guys spend a few months just focusing 100% on bug fixes? Elementor has over 3000 issues unresolved on github alone, and some of them are not so minor and have been around for over a year. Elementor has been feeling more and more buggy (more than it's ever been) over the past 4-5 months and if the team spent more time on QA and bug fixing this could've been avoided. Quality > Quantity in terms of features. My team has built close to 1000 sites with Elementor over the past couple years and we are currently researching other options because of the current state of Elementor.

**Comment by loucasl** [comment]
This is exactly what Elementor is doing in the 3.16 beta:

https://github.com/orgs/elementor/discussions/23486

**Comment by torchsmith** [comment]
I see improvements on there and some bug fixes, but I mean 100% bug fixing, zero features/improvements (unless the improvements are inherent from the bug fixing) over 3000 issues is crazy.

**Comment by simedwards** [comment]
elementor just upgraded to 3.16 and i have lost quode addons widgets ! notably the portfolio list widget which is basically my whole front page !
i have reverted to 3.15 to solve the problem ... for now ...
